frappe_s3_attachment/controller.py

```diff
@@ -51,6 +51,19 @@ def key_generator(self, file_name, parent_doctype, parent_name):
         """
         Generate keys for s3 objects uploaded with file name attached.
         """
+        hook_cmd = frappe.get_hooks().get("zerodha_s3_key_generator")
```

Comment:
can we rename  the hook to just s3_key_generator

Reply:
@shridarpatil udated 👍